flake.lock

```diff
@@ -187,7 +187,6 @@
     "root": {
       "inputs": {
         "crane": "crane",
-        "floco": "floco",
```

Comment:
non-blocking: the floco lock entry is still there. Either:
- leave it, let the next big update take care of it
- remove manually